conf/devel.yaml

```diff
@@ -1,13 +1,91 @@
 ---
-stenographer_instances:
-    - { host: 127.0.0.1, sensor: sensor-001, port: 1234, key: /etc/stenographer/certs/client_key.pem, cert: /etc/stenographer/certs/client_cert.pem, ca: /etc/stenographer/certs/ca_cert.pem }
+# Configure your stenographer instances here. sensor is a just an arbitrary unique name
+STENOGRAPHER_INSTANCES:
+    - { host: 127.0.0.1, sensor: sensor-1, port: 1234, key: /etc/stenographer/certs/client_key.pem, cert: /etc/stenographer/certs/client_cert.pem, ca: /etc/stenographer/certs/ca_cert.pem }
 
 # Flask config values
 DEBUG: true
 TESTING: true
-SECRET_KEY: CHANGE_THIS_SECRET_KEY_TO_RANDOM_VALUE
+SECRET_KEY: CHANGE_THIS_SECRET_KEY_TO_ANY_RANDOM_VALUE
 SESSION_COOKIE_NAME: DOCKET_SESSION
 USE_X_SENDFILE: false
-LOGGER_NAME: docket
+
+# Celery configuration - CELERY_URL is many things :/
 CELERY_URL: redis://localhost:6379
+# all results are simply written to disk, at the moment there's no need to care about brokering them
+CELERY_TASK_IGNORE_RESULT: true
+
+# Docket configuration
+# SPOOL_DIR     - where captures are saved and served - if you change this make sure to fix the webserver's static file configuration
 SPOOL_DIR: /var/spool/docket/
+# WEB_ROOT      - the base url for all docket requests: 
+# WEB_ROOT=/foo/ => http://HOST:PORT/foo/status, http://HOST:PORT/foo/q/after/5m/host/1.2.3.4
+WEB_ROOT: /
+# DATE_FORMAT   - strftime format for showing a datetime to a user, not used yet
+DATE_FORMAT: '%Y%jT%H:%M:%S'
```

Comment:
Any reason not to simply use ISO3339 (a stricter version of ISO8601)? So, for example `2018-01-01T12:00:27Z`

Reply:
The default can be anything users want.  I chose this simply because our time windows are less than one day and saving space on a few values you already know seemed reasonable.  Also I assume ordinal date is easier to regex.